packages/tools/client-debugger/client-debugger-view/src/test/VisualTree.test.tsx

```diff
@@ -0,0 +1,96 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import React from "react";
+// eslint-disable-next-line import/no-unassigned-import
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import {
+	FluidObjectTreeNode,
+	UnknownObjectNode,
+	VisualNodeKind,
+} from "@fluid-tools/client-debugger";
+import { SharedCounter } from "@fluidframework/counter";
+import { MockFluidDataStoreRuntime } from "@fluidframework/test-runtime-utils";
+
+// eslint-disable-next-line import/no-internal-modules
+import { UnknownDataView } from "../components/UnknownDataView";
```

Comment:
You'll want to add these exports to `components/index` so you can import from the directory and avoid having to disable the linter rules.